src/coreComponents/physicsSolvers/wavePropagation/pygeosx/mesh.py

```diff
@@ -0,0 +1,297 @@
+'''
+Created on 18/02/2021
+
+@author: macpro
+'''
+import numpy as np
+import pygeosx
+import sys
+from mpi4py import MPI
+import numpy as np
+from functions import *
+    
+
+def initialize_pyMesh(problem):
+    """Get mesh informations from GEOSX mesh for the seismic acquisition
```

Comment:
This should not be used for any reasonably large mesh, I imagine it will be quite slow.

Reply:
The initialize_pyMesh() should not be used for later simulation. It is currently usefull to create our seismic acquisition based on the mesh. We will later use a .sgy file already containing all the informations we need for the seismic acquisition.